apps/storybook/src/Table.test.ts

```diff
@@ -20,7 +20,7 @@ describe('Table', () => {
     'Global Filter',
     'Initial State',
     'Full',
-    'Full 2',
+    'Full2',
```

Comment:
This was an intended change? Does Ladle treat these names differently, thus requiring this change?

Reply:
yes, storybook automatically adds a space, whereas ladle doesn't.

notice the extra "-" in the storybook url
- storybook: https://itwin.github.io/iTwinUI/react/?path=/story/core-table--full-2
- ladle: https://itwin.github.io/iTwinUI/1669/react/?story=core--table--full2